consensus/istanbul/core/vrank.go

```diff
@@ -85,19 +97,22 @@ func (v *Vrank) HandleCommitted(blockNum *big.Int) {
 		return
 	}
 
-	committedTime := v.TimeSinceStart()
-	if v.threshold > committedTime {
-		v.threshold = committedTime
-	}
-
-	vrankQuorumCommitArrivalTimeGauge.Update(int64(committedTime))
 	if len(v.commitArrivalTimeMap) != 0 {
 		sum := int64(0)
-		for _, v := range v.commitArrivalTimeMap {
-			sum += int64(v)
+		quorumCommitTime := time.Duration(0)
```

Comment:
how about moving `v.firstCommit` assignment here? like
```
	firstCommitTime := time.Duration(math.MaxInt64)
	quorumCommitTime := time.Duration(0)
	for _, arrivalTime := range v.commitArrivalTimeMap {
		sum += int64(arrivalTime)
		if firstCommitTime > arrivalTime {
			firstCommitTime = arrivalTime
		}
		if quorumCommitTime < arrivalTime {
			quorumCommitTime = arrivalTime
		}
	}
	v.firstCommit = firstCommitTime
	v.quorumCommit = quorumCommitTime
```

Reply:
fixed in https://github.com/klaytn/klaytn/pull/1919/commits/ef507bc4e21767d0de9db9ad415710c55ac88f78